Resources/Prototypes/Recipes/Lathes/security.yml

```
@@ -656,4 +656,4 @@
   materials:
      Steel: 150
      Plastic: 100
-     Glass: 20
+     Glass: 20
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)